src/js/UnifileService.js

```diff
@@ -124,9 +124,9 @@ export default class UnifileService {
     });
   }
 
-  cd (path) {
+  cd (path, preventAuth=false) {
```

Comment:
in fact  some services won't require auth, but this is about another topic here: when CE starts, if the path is "/" it will list services, and if there is only one service we want to `cd` into this folder. But without auth because we don't want a popup to be opened and blocked (not in a onclick). What happen here is we `cd` the service and if the user is not logged in it will return a `403 forbidden` and we will be back to "/"

The best thing would be to send a `isLoggedIn` field with the services, so if the user is loggedin we `cd` in it, and when we `cd` a service to which we are logged in, CE should not bother opening a popup for auth. I made a PR for that but I think it died forgotten.. Maybe it was wrong :-/